Shipr-Helm/templates/redis-storageclass.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ .Values.redis.storageClassName }}
+provisioner: {{ .Values.redis.provisioner }}
+reclaimPolicy: {{ .Values.redis.reclaimPolicy }}
+volumeBindingMode: {{ .Values.redis.volumeBindingMode }}
+parameters:
+  type: {{ .Values.redis.parameters.type }}
+  fsType: {{ .Values.redis.parameters.fsType }}
+  encrypted: {{ .Values.redis.parameters.encrypted }}
+mountOptions:
+  - {{ .Values.redis.mountOptions.option1 }}
+  - {{ .Values.redis.mountOptions.option2 }}
```

Comment:
Poor practice to list options like this, it can be improved.

Reply:
Done